.github/workflows/release-image.yml

```
@@ -0,0 +1,37 @@
+    branches:
+      - main
+
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set tags
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:106: Double quote to prevent globbing and word splitting

(shellcheck)